crates/meilisearch/tests/index/update_index.rs

```diff
@@ -106,15 +110,16 @@ async fn error_update_existing_primary_key() {
 
 #[actix_rt::test]
 async fn error_update_unexisting_index() {
-    let server = Server::new().await;
-    let (task, code) = server.index("test").update(None).await;
+    let server = Server::new_shared();
+    let index = server.unique_index();
+    let (task, code) = index.update(None).await;
 
     assert_eq!(code, 202);
 
```

Comment:
Same here, we could use the special nonexistent index that already exists instead of making a unique index